Ensure that the generated package.json and tsconfig.json end with a newline. (#12186)
withastro/astro: .changeset/tidy-bugs-yawn.md

---
'create-astro': minor
---

> The generated package.json and tsconfig.json will have a newline at the end of the file.

Reviewer: ```suggestion
Ensure new line at the end of the generated package.json and tsconfig.json files
```
